### PR TITLE
rename column date to timestamp. new column newdate to 'Date'. Format…

### PR DESCRIPTION
**Pull request by eoliveros**
… newdate to DateTime. New restricted model view for TokenTx. Use bootstrap modal.

This appears to work.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -612,6 +612,94 @@ class TopicModelView(RestrictedModelView):
...
+        attachment = json_obj["attachment"]
+        signature = json_obj["signature"]
+        #txtype = json_obj["type"] ###can't use this
+
+        html = '''
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#Modal{}">
+        Tx Details
+        </button>
+
+<div class="modal fade" id="Modal{}" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel1" aria-hidden="true">
```
Comment (djpnewton) [inline models.py]: `id="Modal{}"` should probably be `id="TxDetailsModal{}"` so it doesnt class this with the txid modals

File: models.py [hunk 2]
```diff
@@ -612,6 +612,94 @@ class TopicModelView(RestrictedModelView):
...
+        return Markup(html)
+
+    def _format_txid_html(view, context, model, name):
+        ids = model.txid
+        truncate_txids = str(ids[:6]+'...')
+        html = '''
+        <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#Modal{}">
+        {}
+        </button>
+<div class="modal fade" id="Modal{}" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```
Comment (djpnewton) [inline models.py]: `id="Modal{}"` should probably be `id="TxidModal{}"` so it doesnt class this with the tx details modals

File: models.py [hunk 3]
```diff
@@ -612,6 +612,94 @@ class TopicModelView(RestrictedModelView):
...
+        assetId = json_obj["assetId"]
+        feeAssetId = json_obj["feeAssetId"]
+        senderPublicKey = json_obj["senderPublicKey"]
+        recipient = json_obj["recipient"]
+        amount = json_obj["amount"]/100
+        fee = json_obj["fee"]/100
+        timestamp = json_obj["timestamp"]
+        attachment = json_obj["attachment"]
+        signature = json_obj["signature"]
+        #txtype = json_obj["type"] ###can't use this
```
Comment (djpnewton) [inline models.py]: why dont you show the type?

File: models.py [hunk 4]
```diff
@@ -612,6 +612,94 @@ class TopicModelView(RestrictedModelView):
...
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+         assetId: {}<br/>
+         feeAssetId: {}<br/>
+         senderPublicKey: {}<br/>
+         recipient: {}<br/>
+         amount: {} ZAP<br/>
```
Comment (djpnewton) [inline models.py]: dont put zap here put the `config['ASSET_NAME']`

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: models.py (hunk 1 above)
Comment (eoliveros) [reply models.py]: changed

File: models.py (hunk 2 above)
Comment (eoliveros) [reply models.py]: changed

File: models.py (hunk 4 above)
Comment (eoliveros) [reply models.py]: changed

File: models.py (hunk 3 above)
Comment (eoliveros) [reply models.py]: added it. it was erroring before because one of the examples i had didnt have the key "type" in it.

File: models.py [hunk 5]
```diff
@@ -612,6 +612,95 @@ class TopicModelView(RestrictedModelView):
...
+        assetId = json_obj["assetId"]
+        feeAssetId = json_obj["feeAssetId"]
+        senderPublicKey = json_obj["senderPublicKey"]
+        recipient = json_obj["recipient"]
+        amount = json_obj["amount"]/100
+        fee = json_obj["fee"]/100
+        timestamp = json_obj["timestamp"]
+        attachment = json_obj["attachment"]
+        signature = json_obj["signature"]
+        txtype = json_obj["type"] ###can't use this
```
Comment (djpnewton) [inline models.py]: get rid of `###can't use this`

**Review by djpnewton — CHANGES_REQUESTED** [review]

**Pull request merged**